programs/zstdcli.c

```diff
@@ -1213,6 +1227,25 @@ int main(int const argCount, const char* argv[])
         if (adaptMin > cLevel) cLevel = adaptMin;
         if (adaptMax < cLevel) cLevel = adaptMax;
 
+        if (showDefaultCParams) {
+            size_t fileNb;
+            static const char* ZSTD_strategyMap[9 + 1] = { "", "ZSTD_fast",
+                "ZSTD_dfast", "ZSTD_greedy", "ZSTD_lazy", "ZSTD_lazy2", "ZSTD_btlazy2",
+                "ZSTD_btopt", "ZSTD_btultra", "ZSTD_btultra2"};
+            for (fileNb = 0; fileNb < (size_t)filenames->tableSize; fileNb++) {
+                const size_t fileSize = (size_t)UTIL_getFileSize(filenames->fileNames[fileNb]);
+                const size_t cParamsIdx = (fileSize < 16 KB) ? 3 : ( (fileSize < 128 KB) ? 2 : ( (fileSize < 256 KB) ? 1 : 0) );
+                ZSTD_compressionParameters cParams = ZSTD_defaultCParameters[cParamsIdx][cLevel];
```

Comment:
There is a function for this, `ZSTD_getCParams()`. We don't need to move the `ZSTD_defaultCParameters` structure out of `zstd_compress.c`.